src/api/spec/models/package_spec.rb

```diff
@@ -448,4 +448,22 @@
       it { is_expected.to be_falsey }
     end
   end
+
+  describe '#jobhistory_list' do
+    let(:backend_url) { "#{CONFIG['source_url']}/build/#{home_project}/openSUSE_Tumbleweed/x86_64/_jobhistory?package=#{package}" }
+
+    subject { package.jobhistory_list(home_project, 'openSUSE_Tumbleweed', 'x86_64') }
+
+    context 'when response is successful' do
+      before { stub_request(:get, backend_url) }
+
+      it { is_expected.to be_truthy }
```

Comment:
I would read such a test as some kind of specification. Right now it would mean that jobhistory_list returns a boolean. But I suspect it's actually a hash with a set of data.

Please also test that the returned data matches with what we expect.